config.py

```diff
@@ -84,15 +85,17 @@ def build_db_uri(
         db_endpoint_arg=DB_ENDPOINT,
         db_name_arg=DB_NAME,
```

Comment:
If we are not using these variables, then why they are used in function definition?

Reply:
this variables is being used for the linux/windows os, by uncommenting line 131 below and comment out line 129

Reply:
this variables is being used for the linux/windows os, by uncommenting line 131 below and comment out line 129